tests/Credentials/CredentialProviderTest.php

```diff
@@ -1813,19 +1817,27 @@ public function testCachesCacheableInDefaultChain()
             'process_credentials',
             'process_config',
             'ecs',
+            'ecs_cloudshell',
```

Comment:
We should remove this entry, as `$cacheable` here represents the default chain.  `ecs_cloudshell` credentials should still be categorized under `ecs` since it's using the same provider

Reply:
### ok, how do you suggest we test the needs of 2 different scenario whereas
Scenario 1: AWS_CONTAINER_CREDENTIALS_RELATIVE_URI
Scenario 2: Without the above ENV variable, but with AWS_CONTAINER_CREDENTIALS_FULL_URI & AWS_CONTAINER_AUTHORIZATION_TOKEN

Reply:
Removed ecs_cloudshell unit-testing from defaultChain